02-express-tutorial/app.js

```diff
@@ -1,8 +1,41 @@
 const express = require('express')
 const app = express()
-const { products } = require("./data");
+const { products } = require("./data")
+const peopleRouter = require("./routes/people")
+
+const logger = (req, res, next) => {
+    time = new Date()
+    console.log(`Method: ${req.method} URL: ${req.url} Time: ${time.getHours()}:${time.getMinutes()}:${time.getSeconds()}`)
+    next()
+}
+
+app.use('/', logger)
```

Comment:
Instead of passing a single route for the logger middleware, use it for every route:

```suggestion
app.use(logger)
```